lib/serverless_helpers.rb

```diff
@@ -11,4 +11,9 @@ class Error < StandardError; end
 
   extend Dry::Configurable
   setting :event_parser, Event::Parser::Plain.new
+
+  setting :provider do
+    setting :identifier, 'aws'
+    setting :type, 'sns'
```

Comment:
@abhi4nfp This looks AWESOME.  However, this feels like a required configuration; something that should not have a default.  

Also, how do you feel about calling this `event.provider`, and `event.type` ?  In which case, I think `event.parser` would be a good fit as well.